src/rtccertificate.cc

```diff
@@ -25,17 +25,27 @@ static const char kExpires[] = "expires";
 static const char kFingerprints[] = "fingerprints";
 static const char kAlgorithm[] = "algorithm";
 static const char kValue[] = "value";
+static const char kPrivateKey[] = "privateKey";
+static const char kCertificate[] = "certificate";
+static const char kFromPEM[] = "fromPEM";
+static const char kToPEM[] = "toPEM";
 
 NAN_MODULE_INIT(RTCCertificate::Init) {
   Local<FunctionTemplate> ctor = Nan::New<FunctionTemplate>(New);
   ctor->SetClassName(LOCAL_STRING(sRTCCertificate));
   ctor->InstanceTemplate()->SetInternalFieldCount(1);
 
+  Nan::SetMethod(ctor, kFromPEM, FromPEM);
```

Comment:
Should be `LOCAL_STRING(kFromPEM)`, I remember having a lot of trouble by forgetting this.